relay-dynamic-config/src/project.rs

```diff
@@ -42,6 +42,10 @@ pub struct Retentions {
     /// This will determine when they are removed from storage.
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub span: Option<RetentionSettings>,
+    /// Retention settings for metrics.
+    /// This will determine when they are removed from storage.
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub metric: Option<RetentionSettings>,
```

Comment:
Going to handle this on the getsentry side[ LOGS-399](https://linear.app/getsentry/issue/LOGS-399/add-retention-config-to-getsentry-before-we-start-let-people-send-data)